gnovm/stdlibs/time/timezoneinfo.gno

```diff
@@ -63,6 +63,14 @@ const (
 	omega = 1<<63 - 1 // math.MaxInt64
 )
 
+// ChainTZ represents the timezone of the chain.
+var ChainTZ *Location = &chainLoc
```

Comment:
What about simply naming it `Local`? The `time.Local` function was removed, which will cause issues. If we want to establish a notion of "local for the user," we need to approach it from the gnoweb (and other clients) perspective. This will ensure the expected timeshift occurs automatically between chain.local and computer.local.

Reply:
Agree & fix: [6aab885](https://github.com/gnolang/gno/pull/3193/commits/6aab8851d32c9d90d60fbc76c8e5e1f6debb6fe2)